apps/code-examples/src/app/code-examples/datetime/datepicker/basic/demo.component.ts

```diff
@@ -13,6 +13,8 @@ import {
 import { SkyDatepickerModule } from '@skyux/datetime';
 import { SkyInputBoxModule } from '@skyux/forms';
 
+import { isDate } from 'moment';
```

Comment:
We shouldn't expose `moment` to the code examples because it's an internal detail. It's not perfect, but for the sake of the code example, could we use `date instanceof Date` instead?

Reply:
Also, this is fixing a bug with our code examples, which is part of the public API, so I'd make the PR title start with `fix(code-examples): `.